Tests/SwiftDocCTests/Utility/Collection+ConcurrentPerformTests.swift

```diff
@@ -57,13 +57,10 @@ class CollectionConcurrentPerformTests: XCTestCase {
         }
         
         // Verify that each next block has started after the previous block has finished.
-        XCTAssertTrue(
-            zip(results, results.dropFirst())
-                .allSatisfy({ result, next -> Bool in
-                    return next.start > result.end
-                }),
-            "Blocks didn't run serially"
-        )
+        for (result, next) in zip(results, results.dropFirst()) {
```

Comment:
I'd be really surprised if that changes anything. By the time this code runs all the results are already finalized - this code is just comparing them. I'd rather save the results into a log any time this assert fails on Linux and try to examine the logs to investigate why the test is failing. Just my 2￠